python/GangaLHCb/Lib/Applications/GaudiExecUtils.py

```diff
@@ -99,7 +99,6 @@ def prepare_cmake_app(myApp, myVer, myPath='$HOME/cmtuser', myGetpack=None):
     full_path = expandfilename(myPath, True)
     if not path.exists(full_path):
         makedirs(full_path)
-        chdir(full_path)
```

Comment:
I think we want to keep the chdir in - it should probably be in the other parts too. After the lb-dev there is the option for a getpack which needs to be run from the lb-dev directory.

As an aside the getpack should probably be changed to git lb-use and git lb-checkout although that might be a bit more involved so we can save that for another PR.